CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required (VERSION 2.6)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules")
 set(ARAROOT_INSTALL_PATH $ENV{ARA_UTIL_INSTALL_DIR})
-#set(CMAKE_CXX_STANDARD 11) #you may also have to specify that you want C++11 standard
+set(CMAKE_CXX_STANDARD 11) #you may also have to specify that you want C++11 standard
```

Comment:
Since you're commenting this out, this needs to be noted on the installation guide. I can't see whether you already did that

Reply:
Ah, so this is actually me uncommenting something! It will by default make AraRoot compile with C++11, instead of whatever it's defaulting to now. I would hope this is uncontroversial at this point?

Reply:
I think many features don't work without C++11. This is only a problem if there are still some ancient EL6 clusters that default to gcc 4.4 or 4.5 or something.

Reply:
Yeah, so, I think at this point, it should be safe. I think we've mostly evolved past running our software on such old systems. In fact, I'm not sure ROOT6 will work without C++ 11...

Reply:
The cvmfs build scripts manually uncomment this line anyway, so this change seems fine to me. It can always be re-commented by individuals if it breaks something for them

Reply:
Okay, great, I'm going to close this convo then!